analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -74,6 +76,32 @@ protected override State ProcessBlock(BasicBlock block)
         return ret;
     }
 
+    private static void LiveCaptureAnalysis(ControlFlowGraph cfg)
+    {
+        var allOperations = cfg.Blocks
+            .Where(x => x.IsEnclosedIn(ControlFlowRegionKind.LocalLifetime))
+            .SelectMany(x => x.OperationsAndBranchValue)
+            .SelectMany(x => x.DescendantsAndSelf())
+            .ToExecutionOrder();
+
+        foreach (var operation in allOperations)
+        {
+            if (IFlowCaptureOperationWrapper.IsInstance(operation.Instance))
+            {
+                var flowCapture = IFlowCaptureOperationWrapper.FromOperation(operation.Instance);
+                if (!IFlowCaptureReferenceOperationWrapper.IsInstance(flowCapture.Value))
+                {
+                    TryAdd(FlowCaptureOperations, flowCapture.Id, [flowCapture.Value]);
```

Comment:
What is stopping us from targetting netstandard2.1 so that TryAdd is available? (I guess compatibility with netframework?)